docs/DeveloperGuide.md

```diff
@@ -957,7 +957,7 @@ To address this, we plan to make the Duplicate Check for Events case sensitive,
 ### 4.6 Input Validation for Patient's Name
 
 Presently, the name of patient, referred to as `NAME`, do not have sufficient input validation for the validity of the `NAME`.
-It currently uses regex expression `[\\p{Alnum}][\\p{Alnum} ]*` to validate the input for patient's name.
+It currently uses regex expression `[\\p{Alnum}][\\p{Alnum} ]*` where it is a String that consists of alphanumeric characters and spaces, to validate the input for patient's name.
```

Comment:
Perhaps could be more specific of who is **it** as **it** is quite ambiguous! Could consider rephrasing to: ~It~ PatientSync currently uses regex expression `[\\p{Alnum}][\\p{Alnum} ]*` to check that the user input for `NAME` is a String that consists of only alphanumeric characters and spaces, to validate the input for patient's name.

Reply:
I get what you mean. This sounded more detailed 👍🏻